src/GovUk.Education.ExploreEducationStatistics.Admin/Services/DataBlockService.cs

```diff
@@ -31,20 +34,26 @@ public class DataBlockService : IDataBlockService
         private readonly IReleaseContentBlockRepository _releaseContentBlockRepository;
         private readonly IUserService _userService;
         private readonly IMapper _mapper;
-
+        private readonly IBlobCacheService _cacheService;
+        private readonly ICacheKeyService _cacheKeyService;
+        
         public DataBlockService(ContentDbContext context,
             IPersistenceHelper<ContentDbContext> persistenceHelper,
             IReleaseFileService releaseFileService,
             IReleaseContentBlockRepository releaseContentBlockRepository,
             IUserService userService,
-            IMapper mapper)
+            IMapper mapper, 
+            IBlobCacheService cacheService, 
```

Comment:
I think this might need wiring up with a dedicated `BlobStorageService` that has a connection key string for `CoreStorage`, and the same in other services that are invalidating private cache items using `BlobStorageService` like `TopicService`. I'm not 100% sure though as I can't see where it's currently getting candidates of IBlobCacheService to autowire in `Startup` at all right now.

Reply:
> 

Sorry, just seen that you added this in your PR

`services.AddTransient<IBlobCacheService, BlobCacheService>();`

and combined with this that already existed explains how it's working

`services.AddTransient(provider => GetBlobStorageService(provider, "CoreStorage"));`